README.md

```diff
@@ -30,7 +30,7 @@ or alternatively you can copy and paste the commands in
 
 - Github: [@Smart1-hub](https://github.com/Smart1-hub)
 - Twitter: [@ademola_adebayo](https://twitter.com/ademola_adebayo)
-- LinkedIn: [Ademola Adebayo](https://www.linkedin.com/in/ademola-adebayo-81051578/)
+- LinkedIn: [Ademola Adebayo](https://www.linkedin.com/in/ademola-adebayo)
 
```

Comment:
- [ ] [Optional] It will be helpful if you add how to run tests locally in the `Getting Started` section in your readme file.

Other than that you both made a great job with testing.